### PR TITLE
GlobalContiner

### PR DESCRIPTION
**Pull request by SoloPashaS**
GlobalContainert added:)))))))))))

### CODE REVIEW

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+using System.Threading.Tasks;
+
+namespace DBlab2
+{
+    static class GlobalContainer
+    {
+        private static Dictionary<string, string[]> DictOfLists = new Dictionary<string, string[]>
+        {
+            ["Author"] = new string[] { "FirstName", "LastName" },
+            ["Book"] = new string[] { "Title", "Year", "PublisherId", "Publisher" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:14]: - Publisher

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            ["BookAuthor"] = new string[] { "BookId", "AuthorId", "Book", "Author" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:15]: - Book

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            ["Cathedra"] = new string[] { "FacultyId", "Name", "Faculty" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:16]: - Faculty

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            ["Faculty"] = new string[] { "Name" },
+            ["Group"] = new string[] { "Number", "SpecialityId", "CathedraId", "Cathedra", "Speciality" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:18]: - Cathedra
- Specialty

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            ["LibraryContext"] = new string[] { "Specialities", "Faculties", "Groups", "Cathedras",
+                "Students", "Authors", "Books", "BookAuthors", "Publishers", "StudentCards", "Teachers", "Positions",
+                "TeacherCards", "LibraryEmployees"},
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:21]: Take this off.

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+            ["LibraryEmployee"] = new string[] { "FirstName", "LastName", "PositionId", "Position" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:22]: - Position

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+            ["Position"] = new string[] { "PosName" },
+            ["Program"] = new string[] { "" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:24]: Take this off.

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+            ["Publisher"] = new string[] { "Name" },
+            ["Speciality"] = new string[] { "Name", "Number", "FacultyId", "Faculty" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:26]: - Faculty

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+            ["Student"] = new string[] { "FirstName", "LastName", "Year", "GroupId", "Group" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:27]: - Group

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+            ["StudentCard"] = new string[] { "StudentId", "TakenDate", "DueDate", "ReturnedDate",
+                "BookId", "LibraryEmployeeId", "Student", "Book", "LibraryEmployee" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:29]: - Book
- Student
- LibraryEmployee

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+            ["Teacher"] = new string[] { "FirstName", "SecondName", "CathedraId", "Cathedra" },
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:30]: - Cathedra

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+            ["TeacherCard"] = new string[] { "TeacherId", "TakenDate", "ReturnedDate", "LibraryEmployeeId",
+                "BookId", "Teacher", "LibraryEmployee", "Book"}
```
Comment (ArthurMamedov) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:32]: - Teacher
- LibraryEmployee
- Book

**Review by ArthurMamedov — CHANGES_REQUESTED** [review]

File: DBLab2/DBLab2/Common/GlobalContainer.cs [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DBlab2
+{
+    static class GlobalContainer
+    {
+        private static Dictionary<string, string[]> DictOfLists = new Dictionary<string, string[]>
```
Comment (AlexValder) [inline DBLab2/DBLab2/Common/GlobalContainer.cs:11]: `public static`

**Review by ArthurMamedov — APPROVED** [review]
Nice done.

**Pull request merged**